ioSockets.js

```diff
@@ -0,0 +1,53 @@
+// Third-party dependencies
+const express = require('express')
+const http = require('http')
+const socket = require('socket.io')
+
+// In-house dependencies
+const db = require('./db/db.js')
+
+// Setup Socket
+const app = express()
+const io = socket(http.Server(app))
+let locationsData
+
+function connect(locations) {
```

Comment:
Need to test that this whole part works.

Reply:
Actually, @mariocimet , when you have a chance, can we talk a bit more about how this is supposed to work. I'm wondering if maybe we can just move the `locations` variable into this file and take it out of `index.js` entirely. But when searching the code, I don't see anywhere where the value of `locations` changes. How is this supposed to work? Thanks!

Reply:
I think we discussed this in the meeting - it looks like this locations variable can go